kamon/README.md

```diff
@@ -0,0 +1,147 @@
+# Kamon 
+
+New Relic [provides two reporters](https://github.com/newrelic/kamon-newrelic-reporter) 
+for [Kamon telemetry](https://kamon.io/):
+* `NewRelicMetricsReporter`
+* `NewRelicSpanReporter`
+
+These reporters leverage the Kamon reporter/module API and integrate with
+data from the Kamon [kanela agent](https://github.com/kamon-io/kanela).
+
+---
+# Spans
+
+This section describes Kamon spans that are sent to the 
+[New Relic span ingest API](https://docs.newrelic.com/docs/understand-dependencies/distributed-tracing/trace-api/introduction-trace-api).
+
+## Common attributes
+
+All spans will have the following common attributes on them:
+* `instrumentation.source`: "kamon-agent"
+* `service.name`: value comes from the config at `kamon.environment.service`
+
+## Span field data
+
+Other span field data maps over pretty directly.  The field on the left is the new relic
+span field name, and the right is where it is sourced:
+
+* `spanId`: `kamonSpan.id.string`
```

Comment:
maybe write this as just <kamon provided span id>? With the backticks, it makes it look like a literal